api/src/main/java/com/epam/esm/domain/Certificate.java

```diff
@@ -1,4 +1,28 @@
 package com.epam.esm.domain;
 
-public class Certificate {
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Certificate implements Serializable {
+
+    private static final long serialVersionUID = -6102006421982197342L;
+    private Long id;
+    private String name;
+    private String description;
+    private LocalDateTime createDate;
```

Comment:
NIT: I feel like Instant should be used here, but this is a point for discussion.